direwolf.c

```diff
@@ -1246,6 +1251,7 @@ static void usage (char **argv)
 	dw_printf ("    -c fname       Configuration file name.\n");
 	dw_printf ("    -l logdir      Directory name for log files.  Use . for current.\n");
 	dw_printf ("    -r n           Audio sample rate, per sec.\n");
+	dw_printf ("    -m             Use metric instead of US customary units.\n");
```

Comment:
I would be more politically neutral:
```suggestion
	dw_printf ("    -m             Use metric instead of imperial units.\n");
```

Reply:
"US customary" is an actual term. And it's more accurate since there's only Myanmar, Liberia, and USA who are not officially metric (and the first two are almost done migrating by now, it seems).

It doesn't really matter here that imperial gallon and US customary gallon are not the same, since for "feet" they're the same. So for "feet" I guess "imperial" is technically correct, even if not historically.

But I'll push a new commit soon changing to "imperial", if you prefer.